$MirOS: ports/games/xbill/patches/patch-Game_cc,v 1.1 2006/12/30 22:56:58 tg Exp $
--- Game.cc.orig	Thu Jan 23 16:22:30 1997
+++ Game.cc	Tue Sep 21 20:57:59 2010
@@ -10,7 +10,11 @@ Game game;
 UI ui;
 
 int Game::RAND(int lb, int ub) {
-	return (rand()%(ub-lb+1) + lb);
+#ifdef arc4random_uniform
+	return (arc4random_uniform(ub-lb+1) + lb);
+#else
+	return (arc4random()%(ub-lb+1) + lb);
+#endif
 }
 
 int Game::MAX(int x, int y) {
@@ -200,7 +204,6 @@ void Game::main(int argc, char **argv) {
 			case 'l':
 			case 'L': level = MAX (1, atoi(optarg)); break;
 		}
-	srand(time(NULL));
 	ui.make_mainwin();
 	ui.graph_init();
 	ui.clear();
